RoboticonColony/Assets/Code/RoboticonFactory.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// The Roboticon Factory. Handles interactions between all roboticons and the player / market.  
+/// </summary>
+sealed class RoboticonFactory
+{
+
+    /// <summary>
+    /// customisationsList is a list of all RoboticonCustomisations
+    /// </summary>
+    private List<RoboticonCustomisation> customisationsList;
+
+    /// <summary>
+    /// RoboticonList is a list of all the roboticons
+    /// </summary>
+    public List<Roboticon> RoboticonList { get; private set; }
+
+    /// <summary>
+    /// Creates a Roboticon Factory.
+    /// </summary>
+    public RoboticonFactory()
+    {
+        customisationsList = null;
+        RoboticonList = null;
+
+        /// Temporary initialisation of customisations, may be done through reading in a file later. 
+        createCustomisations("Ore 1", 2, null, ItemType.Ore, 10);
+        createCustomisations("Power 1", 2, null, ItemType.Power, 10);
+    }
+
+    /// <summary>
+    /// Creates a Roboticon instance that creates a non-customised roboticon in the specified tile.
+    /// </summary>
+    /// <param name="selectedTile"> The Tile that the roboticon is being placed on. </param>
+    public void CreateRoboticon(Tile selectedTile)

```

Comment:
There should only be two ways to get a roboticon:
1. the market can use some of its ore (i think ore, check req doc) to stock up on roboticons.
2. the player can buy roboticons with money
So the createRoboticon method should be private and you should have a private method for the market to exchange ore for roboticons, and a public method for players to buy roboticons with money.